# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1315592141 25200
#      Fri Sep 09 11:15:41 2011 -0700
# Node ID fc0ea95fbba0fb0e68481d191592f5febd18ff5e
# Parent  45263c0bcaa6de41dab0711c34d7849b5ef7572b
__makeinfo__.m: Remove unused third input argument.

* __makeinfo__.m: Remove unused third input argument.

diff --git a/scripts/help/__makeinfo__.m b/scripts/help/__makeinfo__.m
--- a/scripts/help/__makeinfo__.m
+++ b/scripts/help/__makeinfo__.m
@@ -45,20 +45,20 @@
 ## @end example
 ##
 ## @noindent
 ## otherwise.
 ##
 ## The optional output argument @var{status} contains the exit status of the
 ## @code{makeinfo} program as returned by @code{system}.
 
-function [retval, status] = __makeinfo__ (text, output_type = "plain text", see_also = [])
+function [retval, status] = __makeinfo__ (text, output_type = "plain text")
 
   ## Check input
-  if (nargin < 1 || nargin > 3)
+  if (nargin < 1 || nargin > 2)
     print_usage ();
   endif
 
   if (!ischar (text))
     error ("__makeinfo__: first input argument must be a string");
   endif
 
   if (!ischar (output_type))
